ast/ast.go

```diff
@@ -283,11 +284,11 @@ type (
 
 	PutProc struct {
 		Node
-		Clauses []PutClause `json:"clauses"`
+		Clauses []Assignment `json:"clauses"`
```

Comment:
If you're going to rename this type, please finish the job by renaming the fields and variables of the type, either in this PR or in another. Leaving the clause terminology in place is confusing.

Reply:
Ok. How far would you like this to go: all the way including renaming things like  https://github.com/brimsec/zq/blob/master/proc/put.go#L39 ?

Reply:
(I'll do this in a separate PR upon answer to the above question).